src/docs/sdk/event-payloads/contexts.mdx

```diff
@@ -375,3 +375,30 @@ The following example illustrates the contexts part of the <Link to="/sdk/event-
   }
 }
 ```
+
+## Culture Context
+
+Culture Context describes certain properties of the culture in which 
+the software is used.
+
+The `type` and default key is `"culture"`.
+
+`calendar`
+
+: _Optional_. For example `GregorianCalendar`. Free form string. 
+
+`display_name`
+
+: _Optional_. Human readable name of the culture. For example `English (United States)`
```

Comment:
`display_name` should probably be inferred from `local`. So if `local` is `en-US`, `display_language` should be `English (United States)`.